submissions/burning-questions/burning-questions.md

```diff
@@ -0,0 +1,382 @@
+
+Now, let us look at the same example with three productivity 3 modules in every machine instead of efficiency 1 modules. Despite increasing the power consumption they also increase the amount of fuel produced, by 10% per machine per module:
+
+{% include image.html
+
+src= 'src'
+ alt='Processing coal into solid fuel with heavy oil cracking, at maximum productivity'
+ caption='Processing coal into solid fuel with heavy oil cracking, at maximum productivity'
+
+%}
+
+We see that with the cumulative effects of 30% extra product output across the process results in more than double the amount of solid fuel as before! This adds up to an extra energy gain of nearly 100 MJ! However, we also see that total power consumption has greatly increased to about 50.6 MJ.  Let's see how all these changes reflect to the energy gain: 
+
+`Energy gain = 185.562 - 40 - 1.5 - 12.982 - 5.9716 - 30.040 - 1.688 = 93.3804 MJ`
+
+`Percent gain = 93.3804 MJ / 40 MJ = 233.451% ≈ 233%`
+
+We find that productivity modules increase energy gains from 103% to 233%, meaning that in this setup the solid fuel being produced has **over triple the energy** as the coal used! 
+
+On the other hand, our machines are working at about half the speed as before, which makes power get consumed for twice as long. Perhaps we can use speed modules to counteract this. Let's try switching to two productivity 3 modules and one speed 3 module for each machine:
+
+{% include image.html
+
+src= 'src'
+ alt='Processing coal into solid fuel with heavy oil cracking, at high productivity & speed'
+ caption='Processing coal into solid fuel with heavy oil cracking, , at high productivity & speed'
+
+%}
+
+Compared to the last setup, we now get less than double the solid fuel but the power consumption has approximately halved. Let's see how it reflects to the energy gain: 
+
+`Energy gain = 146.448 - 40 - 1.5 - 5.775 - 2.3966 - 11.400 - 0.693 = 84.6834 MJ`
+
+`Percent gain = 84.6834 MJ / 40 MJ = 211.7085% ≈ 212%`
+
+Compared to no modules offering 103% gain, this setup offers an excellent gain of 212%, but the full-productivity setup with 233% gain is better. This is because the cumulative fuel gains from the third productivity 3 module outweigh the reduced power consumption from the speed 3 module. 
+
+Instead of replacing the third productivity module with a speed module, we can introduce speed beacons. This would allow us to both keep the maximum productivity and cut down machine power consumptions, although the beacons themselves would introduce additional power consumptions to account for. I estimate that good boosts in energy gains are possible by having each beacon affect multiple machines. However, since calculations involving beacons would include additional factors to consider such as machine counts and the physical layout of the fuel processing area, I prefer to leave that rabbit hole alone.
```

Comment:
```suggestion
Instead of replacing the third productivity module with a speed module, we can introduce speed beacons. This would allow us to both keep the maximum productivity and cut down machine power consumptions, although the beacons themselves would introduce additional power consumptions that would have to be accounted for. I estimate that good boosts in energy gains are possible by having each beacon affect multiple machines. However, since calculations involving beacons would include additional factors to consider such as machine counts and the physical layout of the fuel processing area, I prefer to leave that rabbit hole alone.
```